build-system/tasks/visual-diff/index.js

```diff
@@ -502,12 +508,18 @@ async function snapshotWebpages(percy, browser, webpages) {
 
             // Finally, send the snapshot to percy.
             await percy.snapshot(name, page, snapshotOptions);
-            log('travis', colors.cyan('●'));
+            log('travis', hasWarnings ? colors.yellow('●') : colors.cyan('●'));
```

Comment:
Minor :bike::derelict_house:: I know it was I who suggested using cyan dots instead of green dots because a "passing" test could still have diffs. If you think this has changed enough with the new logging / error detection, maybe switch to green dots for "passing" tests?

Reply:
The test could still have visual diffs even if it passes, so I'd rather leave it as cyan. It's a minor thing anyways, I bet most people won't be looking at the Travis logs unless directed by us